static/templates/uploaded_files_list.hbs

```diff
@@ -19,15 +19,13 @@
     </td>
     <td>{{ size_str }}</td>
     <td class="actions">
-	<span class="edit-attachment-buttons">
-            <a type="submit" href="/user_uploads/{{path_id}}" class="btn no-style" title="{{t 'Download file' }}" id="download_attachment" download>
+        <span class="edit-attachment-buttons tippy-zulip-tooltip" data-tippy-content="{{t 'Download file' }}" aria-label="{{t 'Download file' }}">
+            <a type="submit" href="/user_uploads/{{path_id}}" class="btn" id="download_attachment" download>
```

Comment:
Effect of removing `no-style`:
| Before | After |
| --- | --- |
| ![Screenshot from 2021-05-13 18-59-27](https://user-images.githubusercontent.com/67277428/118132531-6b2b8e80-b41d-11eb-87b4-b7d7a7773e49.png) | ![Screenshot from 2021-05-13 18-51-03](https://user-images.githubusercontent.com/67277428/118132571-74b4f680-b41d-11eb-9f4f-1473127a8056.png) |

This makes this download button consistent with the download button present in [**Settings > Your bots**](https://chat.zulip.org/#settings/your-bots)
![Screenshot from 2021-05-13 19-02-09](https://user-images.githubusercontent.com/67277428/118132919-d8d7ba80-b41d-11eb-994f-dd86439fb6e8.png)